### PR TITLE
rename

### PR DESCRIPTION
**Pull request by heapwolf**

### CODE REVIEW

**Comment by jwerle** [comment]
cc @trevnorris @getify

**Review by jwerle — APPROVED** [review]

**Review by chicoxyzzy — COMMENTED** [review]
macOS and Linux CI are failing because `io` should be changed to `socket-api` in the [shared workflow](https://github.com/socketsupply/socket-io-tests-workflow/blob/main/.github/workflows/tests.yml). Let's make this change first to make sure CI tests pass again.

**Review by chicoxyzzy — APPROVED** [review]

File: bin/repl.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const proc = spawn('ssc', args, {
 })
 
 let nextId = 0
-let socket = null
+let sock = null
```
Comment (getify) [inline bin/repl.js]: I think `sock` and `socket` being different things, in the same code base, may turn out to be a bit confusing for future maintenance/review purposes.

Might we consider `conn` as the variable name for socket connections?

File: bin/repl.js [hunk 2]
```diff
@@ -245,16 +245,16 @@ async function evaluate (cmd, ctx, file, callback) {
   if (!isTry && !/import\s*\(/.test(cmd)) {
     if (/^\s*await/.test(cmd)) {
       cmd = cmd.replace(/^\s*await\s*/, '')
-      cmd = `void Promise.resolve(${cmd}).then((result) => console.log(io.util.format(result)))`
+      cmd = `void Promise.resolve(${cmd}).then((result) => console.log(socket.util.format(result)))`
     } else if (lastName) {
-      cmd = `${cmd}; io.util.format(${lastName});`
+      cmd = `${cmd}; socket.util.format(${lastName});`
```
Comment (getify) [inline bin/repl.js:250]: since it's a proper branded noun, and to make it stand out, might we consider `Socket` instead of `socket`?

Comment (chicoxyzzy) [reply bin/repl.js:250]: @jwerle won't the current change break a REPL?

Comment (heapwolf) [reply bin/repl.js:250]: The repl is already broken for other reasons. Im going to merge this and move forward.

**Pull request merged**